rapids_version.cmake

```diff
@@ -0,0 +1,11 @@
+file(READ "${CMAKE_CURRENT_LIST_DIR}/VERSION" _rapids_version)
```

Comment:
I would like to keep the number of files in the top level of the repository limited. Could we consider making this code a part of `fetch_rapids.cmake`? We could rename the `fetch_rapids.cmake` file to something else like `rapids_config.cmake` if it seems appropriate.